Kitodo/src/main/java/de/sub/goobi/forms/HelperForm.java

```diff
@@ -47,83 +42,13 @@
 public class HelperForm implements Serializable {
     private static final long serialVersionUID = -5872893771807845586L;
     private transient ServiceManager serviceManager = new ServiceManager();
-    private static final Logger logger = LogManager.getLogger(HelperForm.class);
-    static final String MAIN_JSF_PATH = "/pages";
-    private static final String IMAGE_PATH = "/pages/images";
     private static final String CSS_BASE_PATH = "/WEB-INF/resources/css";
     private static final String CSS_PATH = "/userStyles";
 
     public String getVersion() {
         return GoobiVersion.getBuildversion();
     }
 
-    /**
-     * Get application logo.
-     *
-     * @return link to logo
-     */
-    public String getApplicationLogo() {
-        String logo = getServletPathWithHostAsUrl() + IMAGE_PATH + "/template/";
-        logo += ConfigCore.getParameter("ApplicationLogo", "kitodo-header-logo.svg");
```

Comment:
- You should even remove `ApplicationLogo`  entry including the describing comment from all existing `kitodo_config.properties` files.
- If file `kitodo-header-logo.svg` is not used anymore in new design then this file should be removed too.